tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/provider/TieredStorageConfiguration.java

```diff
@@ -47,9 +50,28 @@
     public static final String BLOB_STORE_PROVIDER_KEY = "managedLedgerOffloadDriver";
     public static final String METADATA_FIELD_BUCKET = "bucket";
     public static final String METADATA_FIELD_REGION = "region";
-    public static final String METADATA_FIELD_ENDPOINT = "endpoint";
+    public static final String METADATA_FIELD_ENDPOINT = "serviceEndpoint";
+    public static final String METADATA_FIELD_MAX_BLOCK_SIZE = "maxBlockSizeInBytes";
+    public static final String METADATA_FIELD_READ_BUFFER_SIZE = "readBufferSizeInBytes";
+
     protected static final int MB = 1024 * 1024;
 
+    /*
+     * Previous property names, for backwards-compatibility.
+     */
+    static final String BC_S3_REGION = "s3ManagedLedgerOffloadRegion";
```

Comment:
These probably belong in the test file rather than here, since you are constructing the key from scratch below.